docs/topics/http/middleware.txt

```diff
@@ -359,7 +358,7 @@ both::
     sync/async transitions as possible.
 
     Thus, even if you are wrapping an async view, you may be called in sync
-    mode if there is other, synchronous middleware between you and the view.
+    mode if there is another synchronous middleware between you and the view.
```

Comment:
I feel the comma helped in the original - it's not "another synchronous middleware" in the sense that the middleware is writing is synchronous too, it's "another, synchronous, middleware" in the sense that there is another middleware and, unlike you, it is asynchronous.

Reply:
Yes, reverting